### ISSUE TITLE
Blog: reduce line length

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/covid-19-behind-bars#181]
Feedback from Sasha: reduce line length in the blog

You may need to add columns on each side. If he can make line length around 60 - 70 characters that would be ideal.
https://baymard.com/blog/line-length-readability 

“Here at baymard.com we use Minion Pro in size 20 for our articles. With a 560-pixel wide article container, we end up with about 72 characters per line.”

**Comment by hoshmn** [comment]
hey @sashasashasasha - not quite sure what this should entail. do we want a maximum width for the blog content (so will only shorten line length once it gets to a certain size)? or do we want the content to take up a smaller portion of the width across the board (ideally this would be a number of columns out of 12)?

**Comment by hoshmn** [comment]
@sashasashasasha I capped the length at 700px. This should get us in that general range (it can only be so precise because character width varies, of course). In [this blog post](https://internal-staging--covid-19-behind-bars.netlify.app/blog/lorem-ipsum) I added sections to show how much of the content width is taken up by characters of different widths. We could tweak that 700px value [here](https://github.com/Hyperobjekt/covid-19-behind-bars/blob/sprint-1/packages/gatsby-ucla-site/src/components/blog/blog.js#L129). If there's some way we accomplish this on other sites with more desired results, let me know.
